content/add-organize-data/store-data/streams/streams-add-tags-or-metadata.md

```diff
@@ -0,0 +1,27 @@
+---
+uid: streams-add-tags-or-metadata
+---
+
+# Add tags or metadata
+
+Tags and metadata allow you to associate additional information with a stream. For example, tags can denote special attributes for a stream. Metadata consists of keys and their associated values.
+
+1. From the left pane, select **Data Management** > **Sequential Data Store**.
+
+1. Select a stream.
+
+1. Select **More options** ![More options icon](../../../_icons/default/dots-vertical.svg) > **Edit Stream**.
+
+   A tab opens to allow you to modify the metadata and tags for the stream.
+
+1. Add metadata.
+
+   - Select the **Metadata** tab, and then select **Add Metadata**. In the `Metadata Key` box, enter the name of the key and in the `Metadata Value` box, enter a value for that key.
+
+   - To remove a metadata, select the **X** beside the row to be deleted.
```

Comment:
"remove a metadata" sounds awkward (like saying "remove a data" would). Maybe change to either "To remove metadata" or "To remove a metadata value"